### PR TITLE
[PROGRAMMES-6637] Removed additional clip information.

### PR DESCRIPTION
**Pull request by tomwinnington**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6637

Removed the additional "available until" date information.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/DsShared/Helpers/PlayTranslationsHelper.php [hunk 1]
```diff
@@ -47,9 +47,6 @@ public function translateAvailableUntilToWords(ProgrammeItem $programmeItem, Ser
 
         $translationPrefix = 'iplayer_' . $mediaVerb . '_remaining';
         $text = $this->timeIntervalToWords($remainingTime, false, $translationPrefix);
-        if ($showAvailableUntil && $programmeItem->getStreamableUntil()) {
```
Comment (votemike) [inline src/DsShared/Helpers/PlayTranslationsHelper.php]: Is `$showAvailableUntil` used any more?

Comment (votemike) [reply src/DsShared/Helpers/PlayTranslationsHelper.php]: Instead of deleting this, why not just pass false through instead.

**Comment by tomwinnington** [comment]
Can this be squished?

**Pull request merged**